### DISCUSSION TITLE
Support for List Creation on Right Click

### DISCUSSION THREAD

**Discussion by spacec0de** [outline/outline#2575] [Ideas]
Add lists to the right click menu for easy creation
Workflow:
1. Highlight text area
2. Right click
3. Select UL/OL list to create list

**Comment by tommoor** [comment]
Hey - you mean In the floating formatting toolbar, yea?

**Reply by spacec0de** [reply]
Yes, in the floating toolbar

**Reply by spacec0de** [reply]
Oh - and a hover menu option for plain text, right next to H1 and H2.

This would be to turn whatever element the text currently is (H1, H2, OL, UL, code block, hyperlink, bold, etc) back into plain text.
